### ISSUE TITLE
Search doesn't work for content guide

### ISSUE THREAD

**Issue by mtorres253** [18F/lunr-server#5]
There is valid json here: https://pages.18f.gov/content-guide/search-index.json

When I add this to the config json

``` json
{ "name": "Content Guide",
   "baseurl": "https://pages.18f.gov/content-guide",
   "indexPath": "../content-guide/_site/search-index.json"
}
```

And then try to run this command

`bin/lunr-server lunr-server-config.json`

I get this error

`error launching /Users/michaeltorres/code/lunr-server/bin/lunr-server: failed to parse lunr-server-config.json: watch ENOENT
program terminated`

@mbland filed a pull request a while back (https://github.com/18F/lunr-server/pull/4) that will likely take care of the failure to start the server, but I wonder why it's failing to read that json? Is the indexPath incorrect? 

This seems to be the case now with all the guides, including those already in the config.

**Comment by mbland** [comment]
#4 is for a different error. You're running the server locally on your Mac, ostensibly from the `/Users/michaeltorres/code/lunr-server` directory, whereas the `lunr-server-config.json` file as committed assumes the program will run on the pages.18f.gov server, with its current directory relative to those paths appearing as `indexPath` entries. Unless you've got `/Users/michaeltorres/code/content-guide/_site/search-index.json` present, the program will fail with `ENOENT` (for [no such file or directory](http://man7.org/linux/man-pages/man3/errno.3.html)).

In other words, clone 18F/content-guide into your `/Users/michaeltorres/code` directory, build the site, and try running lunr-server again.

**Comment by mbland** [comment]
Actually, #4 will take care of crash-on-startup, but the real problem here is an understanding of how the program is configured, hence my comment that "#4 is for a different error".

**Comment by mtorres253** [comment]
I'll clone the repo and go from there. Thanks @mbland. We should be releasing everything soon. Will send you links :)
